website/docs/r/cloud_backup_schedule.html.markdown

```diff
@@ -0,0 +1,220 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: cloud_backup_schedule"
+sidebar_current: "docs-mongodbatlas-resource-cloud-backup-schedule"
+description: |-
+    Provides a Cloud Backup Schedule resource.
+---
+
+# mongodbatlas_cloud_backup_schedule
+
+`mongodbatlas_cloud_backup_schedule` provides a resource that enables you to view and modify the snapshot schedule and retention settings for an Atlas cluster with Cloud Backup enabled.  A default policy is created automatically when Cloud Backup is enabled for the cluster.
```

Comment:
This is just copied from the existing - the point of this is to re do this!   This is not just view and modify, it is to create, modify and delete, read - full CRUD.